specification.md

```diff
@@ -5026,7 +5071,124 @@ for your workflow definitions especially in production environments.
 
 To enhance portability when using versioning of your workflow and sub-workflow definitions,
 we recommend using an existing versioning standard such as [SemVer](https://semver.org/) for example.
- 
+
+### Workflow Globals
+
+Workflow globals are used to define static data which is available to [Workflow Expressions](#Workflow-Expressions).
+
+Globals can be defined via the [Workflow top-level "globals" property](#Workflow-Definition-Structure),
+for example:
+
+```json
+"globals": {
+  "Translations": {
+    "Dog": {
+      "Serbian": "pas",
+      "Spanish": "perro",
+      "French": "chien"
+    }
+  }
+}
+```
+
+Globals can only be accessed inside Workflow expressions via the $GLOBALS namespace.
+Runtimes must make globals available to expressions under that namespace.
+
+Here is an example of using globals in Workflow expressions:
+
+```json
+{
+...,
+"globals": {
+  "AGE": {
+    "MIN_ADULT": 18
+  }
+},
+...
+"states":[  
+  {  
+     "name":"CheckApplicant",
+     "type":"switch",
+     "dataConditions": [
+        {
+          "name": "Applicant is adult",
+          "condition": "${ .applicant | .age >= $GLOBALS.AGE.MIN_ADULT }",
+          "transition": "ApproveApplication"
+        },
+        {
+          "name": "Applicant is minor",
+          "condition": "${ .applicant | .age < $GLOBALS.AGE.MIN_ADULT }",
+          "transition": "RejectApplication"
+        }
+     ],
+     ...
+  },
+  ...
+]
+}
+```
+Note that globals can also be used in [expression functions](#Using-Functions-for-Expression-Evaluation),
+for example:
+
+```json
+{
+"functions": [
+  {
+    "name": "isAdult",
+    "operation": ".applicant | .age >= $GLOBALS.AGE.MIN_ADULT",
+    "type": "expression"
+  },
+  {
+    "name": "isMinor",
+    "operation": ".applicant | .age < $GLOBALS.AGE.MIN_ADULT",
+    "type": "expression"
+  }
+]
+}
+```
+
+Workflow globals values should only contain static data, meaning that their value should not 
+contain Workflow expressions. 
+Workflow globals should not have access to [Workflow secrets definitions](#Workflow-Secrets).
+
```

Comment:
Are globals mutable? Can I change them in an expression and if I do, are the new contents also transferred to the next state or synchronized across parallel states when I modify them?

Reply:
This is a good question. I don't think globals should be mutable. Hence maybe we can change to `constants` instead?